package.json

```diff
@@ -5,8 +5,9 @@
   "scripts": {
     "start": "webpack --progress && node browsersync-server",
     "production": "webpack -p --progress && forever start server",
-    "test": "NODE_ENV=test NODE_PATH=. mocha --compilers js:babel/register -r spec/support/setup.mocha.js --recursive 'spec/**/*.test.js' -w",
-    "lint": "eslint ."
+    "test": "NODE_ENV=test NODE_PATH=. mocha --compilers js:babel-core/register -r spec/support/setup.mocha.js --recursive 'spec/**/*.test.js' -w",
+    "lint": "eslint .",
+    "validate": "npm ls"
```

Comment:
It was auto added by git-validate, although I really very hate what it has done.